packages/common-ui/lib/intl/common-ui-de.ts

```diff
@@ -11,33 +11,51 @@ export const COMMON_UI_MESSAGES_GERMAN: Partial<
   add: "Hinzufügen",
   addNew: "Neu hinzufügen",
   agricultureCanada: "Canadische Landwirtschaft",
+  applyButtonText: "Anwenden",
   areYouSure: "Sind Sie sicher?",
+  attachSelected: "Auswahl anhängen",
   attached: "anhängen",
   attributeLabel: "Attribut",
   autoFillGeoSuggestLabel:
     "Standortfelder (automatisch befüllt, Inhalt ist nicht beständig)",
   backToList: "Zurück zur Trefferliste",
+  backToParentFile: "Zurück zur übergeordneten Datei: {parentFilename}",
   backToReadOnlyPage: "Zurück zur schreibgeschützten Seite",
   bulkEdit: "Bearbeitung der Gruppe",
   bulkEditorValidationError: "Fehlerhafte Eingabe in Zeile {row} : {value}",
   bulkSubmissionErrorInfo:
     "fehlerhafte Eingabe: Bitte prüfen Sie die Felder mit roter Kennzeichnung.",
   cancelButtonText: "Annullieren",
+  changesMade: "Änderungen vorgenommen",
```

Comment:
Looks like there is an encoding issue here (there is also other occurrences in the file)